### PR TITLE
Edit AboutUs page

### LINKED ISSUE #3: About Us website field
W7 Task 2

### PR DESCRIPTION
**Pull request by joshuacjx**
Closes #3.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -5,55 +5,67 @@ title: About Us
...
+[[github](http://github.com/josuaaah)]
+[[portfolio](team/josuaaah.md)]
 
 * Role: Developer
-* Responsibilities: UI
+* Responsibilities: Code Quality
+
+### Theodore Leebrant
+
+<img src="images/theodoreleebrant.png" width="200px">
```
Comment (sc-arecrow) [inline docs/AboutUs.md:65]: This image cannot be used, should be replaced with an image of the person or a placeholder image.

**Review by sc-arecrow — CHANGES_REQUESTED** [review]
I found some issues with the individual images

**Review by theodoreleebrant — APPROVED** [review]
LGTM after some minor fixes

File: docs/AboutUs.md (hunk 1 above)
Comment (joshuacjx) [reply docs/AboutUs.md:65]: Have replaced image with a placeholder image.

**Review by sc-arecrow — APPROVED** [review]
LGTM

**Pull request merged**